netd.go

```diff
@@ -339,6 +339,25 @@ func handleContainerStart(netPlugin *plugin.NetPlugin, crt *crt.Crt,
 	return err
 }
 
+func handleContainerStop(netPlugin *plugin.NetPlugin, crt *crt.Crt,
+	contId string) error {
+	// If HOST_GOPATH env variable is set we can assume we are in docker in docker testbed
+	// Here we need to set the network namespace of the ports created by netplugin back to NS of the docker host
+	hostGoPath := os.Getenv("HOST_GOPATH")
```

Comment:
May be we should introduce a more specific environment variable like CONTIV_DIND to avoid any future conflicting use of this variable elsewhere?